tests/test_config.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+from tests.utils import TEST_FIXTURES_DIR
+from torch_audiomentations import from_dict, from_yaml
+from torch_audiomentations import Gain
+
+
+# TODO: uncomment once Compose is available
+# https://github.com/asteroid-team/torch-audiomentations/issues/23
+# from torch_audiomentations import Compose
+
+
+class TestFromConfig(unittest.TestCase):
+    def test_from_dict(self):
+        config = {"Gain": {"min_gain_in_db": -12.0, "mode": "per_channel"}}
+        transform = from_dict(config)
+
+        assert isinstance(transform, Gain)
+        assert transform.min_gain_in_db == -12.0
+        assert transform.max_gain_in_db == 6.0
+        assert transform.mode == "per_channel"
+
+    # TODO: uncomment once Compose is available
+    # https://github.com/asteroid-team/torch-audiomentations/issues/23
+    # def test_from_dict_compose(self):
+    #     config = {
+    #         "Gain": {"min_gain_in_db": -12.0, "mode": "per_channel"},
```

Comment:
Perhaps it's better to be more explicit about how we initialize a Compose? Because Compose is going to have arguments like shuffle. And we are going to have OneOf and SomeOf, which are other ways of applying (several) transforms sequentially

Reply:
IMO, it's probably enough to do that for now and adapt when `Compose` is implemented.

Reply:
@mpariente What do you mean by **"that"** exactly?

Reply:
Sorry 

"It's probably enough to keep it as is for now and adapt when Compose is implemented."

Reply:
So, no change needed from my side for now?

Reply:
We could also keep this PR open until `Compose` is implemented.

Reply:
Yes, let's keep it open until Compose is implemented. I can do that today if I find enough time